condor_watch_q.py

```diff
@@ -988,6 +996,15 @@ def make_table(headers, rows, fill="", header_fmt=None, row_fmt=None, alignment=
         for original_row, processed_row in zip(rows, processed_rows)
     ]
 
+    remaining_columns = terminal_columns - len(lines[0])
+    if progress_bar and remaining_columns > 10:
+        for i, row in zip(range(len(lines)), rows):
+            lines[i] += "".join(make_progress_bar(row, remaining_columns, color=color))
```

Comment:
Love that we get to re-use `make_progress_bar` here - indicates that we got the design right!